### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by Flink91** [skyslide22/blender-addon-for-trackmania-and-maniaplanet#3] [Q&A] [answered]
I am lost on how to actually use this plugin, I think I installed it correctly ( as a Zip file) but nothing shows up. How do I use it? 
Some more documentation would be great.
Cheers Flink

**Comment by skyslide22** [comment, marked as answer]
i have updated the readme.md, please take a look

**Reply by Flink91** [reply]
Yess, very helpful thanks!
